mypy.ini

```diff
@@ -0,0 +1,20 @@
+[mypy]
+python_version=3.8
```

Comment:
should be 3.6

Reply:
Indeed. I temporarily switched to 3.8 to test something and forgot to reset this.